cfi_intro.adoc

```diff
@@ -219,6 +219,27 @@ that do not support the extension or if the extension is not active.
 
 [NOTE]
 ====
+Compilers should flag each object file (for example, using a flag in the elf
+attributes) to indicate if the object file has been compiled with the Zicfisslp
+instructions. The linker should flag the binary/executable generated by linking
+objects as being compiled with the Zicfisslp extension only if all the  objects
+files linked have the same Zicfisslp attribute.
+
+The dynamic loader should enable the use of Zicfisslp extension for a process
+only if all executables (the application and the dependent dynamically linked
+libraries) used by that process have the Zicfisslp attribute. When the use of
+the extension is not enabled for a process then the Zicfisslp instructions in
+that application or in the dynamically linked libraries invoked by that
+process revert to their Zimop/Zccmop defined behavior. This  allows the program
+to functionally execute.
+
+A process that has the Zicfisslp extension enabled may request the dynamic
+loader at runtime to load a new dynamic shared object (using dlopen() for
+example). If the requested object  does not have the Zicfisslp attribute then
```

Comment:
Nit: Extra space

Reply:
Thanks. Addressed in https://github.com/riscv/riscv-cfi/pull/103/commits/ad7f6882ed21e3cd0439b23a98b09fe673b2517b